code/API_definitions/Test_definitions/Traffic_Influence_Test.feature

```diff
@@ -0,0 +1,78 @@
+#/*- ---license-start
+#* CAMARA Project
+#* ---
+#* Copyright (C) 2022 - 2023 Contributors | Deutsche Telekom AG to CAMARA a Series of LF Projects, LLC
+#* The contributor of this file confirms his sign-off for the
+#* Developer Certificate of Origin (http://developercertificate.org).
+#* ---
+#* Licensed under the Apache License, Version 2.0 (the "License");
+#* you may not use this file except in compliance with the License.
+#* You may obtain a copy of the License at
+#*
+#*      http://www.apache.org/licenses/LICENSE-2.0
+#*
+#* Unless required by applicable law or agreed to in writing, software
+#* distributed under the License is distributed on an "AS IS" BASIS,
+#* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#* See the License for the specific language governing permissions and
+#* limitations under the License.
+#* ---license-end
+#*/
+
+@TI @TISanity
+Feature: Automated Traffic Influence API Test
+
+  @TI_Resource_LCM_Mandatory_Parameters_Valid
+  Scenario: Manage Traffic Influence (TI) Resource with mandatory parameters
+    When creating a new TI Resource, with POST, with mandatory parameters ("apiConsumerId", "applicationId")
+    Then it should be created a new TI Resource and the optimal routing will be activated for any user on any location
+    And Response Code is 201 
+    And response contains the TI Resource with the resource identifier ("trafficInfluenceID") valorised with a unique value, the status of the request ("state=ordered") and the previously used parameters valorised as in the POST request
+    When asking for a previously created TI Resource, with GET, with the parameter "trafficInfluenceID" valorised with the reponse of the previous POST
+    Then Response code is 200
+    And response contains a TI Resource with a potentially updated status ("state") reporting the current status of the traffic influece configuration (ordered, created, active, error, deleted)
+	When deleting an existing TI Resource, with DELETE, with the parameter "trafficInfluenceID" valorised with the reponse of the previous POST
+    Then Response Code is 202 and the response message is Accepted meaning that the resource deletion is accepted and in progress. The satus update can be retrived with the GET method on that TI Resource. The final value of the parameter "state" is "deleted".
```

Comment:
Why  202 Accepted? 202 usually indicates an asynchronous action (server is not confirming the deletion, it is confirming the request to delete has been received). If this is a synchronous process then 204 No Content is more appropriate.

Reply:
202 is for resource deletion. the action is indeed asynchronous. The request is accepted and starts a process of network reconfiguration on the UPF for example. the influencing of the traffic is valid until the network is actually reconfigured